examples/ea4tx_remotebox.toml

```diff
@@ -25,7 +25,7 @@ name = "My EA4TX Remotebox"
 # switches with an index < 2 and before all other switched with an index > 2.
 index = 2
 portname = "/dev/ttyACM0"
-
-ipaddress = "192.168.10.109"
-ipport = 6000
-typeConnection=1
\ No newline at end of file
+Ipaddress = "192.168.10.109"
+Ipport = 7011
+# Connection = 0 - Serial; Connection = 1 - tcp/ip 
+Connection=1
```

Comment:
1. please make this parameter lower case. All parameters in the config file are explicitely lower case to avoid confusion and parsing problems.

2. For simplicity for the user and clarity, I suggest to use a string instead of a verbose number. So I would suggest for example `connection=tcp` and `connection=serial`.

Reply:
Hi Tobi
I have changed those variables to lower case and added a prefix (rb) in order to identify those new variables with the module where they are used (RemoteBox)
I think it's better to use rbIpaddress and rbPort instead to share portname as you suggested.
Also changed the ea4tx_remotebox.toml sample file.